tutorials/classical_simulation/Simulating with effective noise channels.ipynb

```diff
@@ -15,25 +15,23 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        config_dephasing = SimConfig(noise="dephasing", dephasing_prob=noise_rate)
</pre><p>Same everywhere on this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/616/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>